### DISCUSSION TITLE
Serverpod cli generate

### DISCUSSION THREAD

**Discussion by Rohithgilla12** [serverpod/serverpod#105] [Q&A] [answered]
Hello, I was searching for documentation on what flags can we pass to the cli and also os there any way to generate only the server folder of I want to use it as a standalone back end server without flutter client and flutter app?

**Comment by vlidholt** [comment, marked as answer]
You can use it as a standalone API server. The protocol is well defined, but currently not yet well documented (especially when it comes to passing serialized objects). Generation of client side code is done in the `myproject_client` directory, so it's safe to just ignore that. The generation of the client code itself is very fast, what takes time is analyzing the server which is required for the server side code anyway.

By running `serverpod help` you will get a list of serverpod commands and the currently supported options.
